src/main/java/org/cascadebot/cascadebot/commands/developer/RestartCommand.java

```diff
@@ -33,7 +33,7 @@ public String command() {
 
     @Override
     public String description() {
-        return "Restarts the bot";
+        return "Restarts the bot via the wrapper.";
```

Comment:
doesn't this just shut it down with the appropriate code? beyond that it doesn't communicate with the wrapper at all.

Reply:
Exactly yeah, but it still does the restarting "via the wrapper" so this description is correct as without the wrapper we couldn't restart.

Reply:
I feel like we could find a better description for this.

Reply:
It's not really important, it's a Dev command anyway

Reply:
fair enough